autogen/main/variables.tf.tmpl

```diff
@@ -96,6 +96,12 @@ variable "http_load_balancing" {
   default     = true
 }
 
+variable "service_externalips" {
```

Comment:
```suggestion
variable "service_external_ips" {
```

Reply:
sure, sounds good - I will make some changes locally and update!